examples/multi-page/page-one.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { Link } from 'react-router-dom';
+import {
+    TextField,
+    EmailField,
+    Page, 
+    DebuggerView
+  } from '@department-of-veterans-affairs/va-forms-system-core';
+
+export default function PersonalInformationPage() {
+    return (
+        <>
+            <Page title="Personal Information" nextPage="page-two">
```

Comment:
With the new approach, users will import the `Page` component into their own react components and build their form pages using va-forms-system-core provided form fields.